src/fondant/pipeline/pipeline.py

```diff
@@ -361,6 +496,8 @@ def add_op(
             "dependencies": dependencies_names,
         }
 
+        return self
```

Comment:
Don't think this is needed since you're not returning the attribute in your read/apply/write functions